test.hxml

```diff
@@ -0,0 +1,14 @@
+## Neko
+-main TestMain
+-lib munit
+-lib hamcrest
+-lib svg
```

Comment:
Not needed, there's `-cp src`, and you may want to test the non-haxelib-set version.